server/auth/registrar.go

```diff
@@ -323,7 +323,7 @@ func (auth *AuthManager) validateFee(conn comms.Link, msgID uint64, acctID accou
 			Code:    msgjson.FeeError,
 			Message: "unsupported asset",
 		}
-		return wait.DontTryAgain
+		return true
```

Comment:
I thought you would want to use `wait.DontTryAgain` instead of a bool with implicit conversion.

Reply:
Am I mixed up about this, or was this change a mistake?

Reply:
Yeah. Must have been a mistake. Fixing.